lib/context/color.ts

```diff
@@ -19,8 +19,10 @@ export type ColorPalette = (
   | 'base'
   | 'accent'
   | 'complement'
+  | 'light'
   | 'lighter'
   | 'lightest'
+  | 'dark'
```

Comment:
this change requires design sign-off

can't we just `[color-palette^="dark"]` in the css? https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors#attrvalue_4

Reply:
I'm ok with switching to the `^` selector.

@coreyvickery is aware these changes were coming, and has been added as a design review here now as well, forgot to do that.

Reply:
> Rather than adding new color palettes

So you are saying don't add `light` and `dark` to the ColorPalette type, but DO add the attribute?  Can you explain that a bit more when you have the chance?

Reply:
`ColorPalette` is a user-facing API, and it represents all the available palettes an element can have. So before, we said that a card could have base, accent, complement, lighter, lightest, darker, or darkest color palettes. That already struck me as too much, but now adding dark and light seems excessive to me. The point of having this type in our project isn't to permit more tests to pass, it's to communicate to developers using RHDS what the high level color theming concept contains. We should be trying to limit this type as much as possible, rather than expanding it to suit our needs as developers.

However, if design is certain that they want users to choose between 4 lightish colour palettes and 5 darkish color palettes when working with our components, well then i suppose this is ok

Reply:
@bennypowers @zeroedin I don't think we need `base`, `accent`, and `complement`.

I do think we need `lightest`, `lighter`, `light`, `darkest`, `darker`, and `dark` though. I can see other design teams using a similar structure, PatternFly might already, but the names might be different. Happy to confirm the color values for each of these as well.

Reply:
If @coreyvickery 's good with it, then I'm good with it :)